src/wp-includes/load.php

```diff
@@ -979,7 +979,9 @@ function wp_get_mu_plugins() {
  * @since 3.0.0
  * @access private
  *
- * @return string[] Array of paths to plugin files relative to the plugins directory.
+ * @return string[] Array of absolute paths to plugin files.
+ *                  Each path is an absolute path to the plugin file,
+ *                  not relative to the plugins directory.
```

Comment:
```suggestion
 * @return string[] Array of absolute paths including WP_PLUGIN_DIR to all active and valid plugin files.
```

While I get your point to stress the difference to the previously wrong doc, this is trippled up information. Imho this change is enough.